api.wordpress.org/public_html/patterns/1.0/tests/test-index.php

```diff
@@ -260,7 +255,7 @@ public function data_search_patterns() {
 
 			// The Core keyword (11) is hardcoded in `test_search_patterns()`, so don't need to specify it here.
 			"only match Core posts" => array(
-				'search_term'       => 'two buttons', // Post ID 727.
+				'search_term'       => 'Two images with text and buttons', // Post ID 727.
```

Comment:
"two buttons" matches a core pattern now.

Reply:
does this cover: https://github.com/WordPress/wordpress.org/pull/393/files#r1792990280?